model_training/trainer/trainer.py

```diff
@@ -110,28 +126,44 @@ def train_and_eval_pipeline(
             train_target = train_dataset.runtime.values
             test_target = test_dataset.runtime
 
-        print(f"Training {model_type} model")
+        logger.info(f"Training {model_type} model")
         with mlflow.start_run(run_name=f"{layer_type}_{model_type}_model"):
+            # MLflow tags
+            repo = f"git@github.com:{self.mlflow_config['dagshub_repo_owner']}/{self.mlflow_config['dagshub_repo_name']}.git"
+            mlflow.set_tags(
+                {
+                    "mlflow.source.git.branch": get_git_branch(),
```

Comment:
Should we record a specific commit together with the branch? Branch pointer will eventually move or be deleted